src/Lecture3.hs

```diff
@@ -246,4 +294,6 @@ Just [8,9,10]
 [8,20,3]
 
 -}
-apply = error "TODO"
+apply :: Functor f => a -> f (a -> b) -> f b
+-- @flip flap@ is a fun expression, plus, I already have the relude dependency, might as well use it
+apply = flip flap
```

Comment:
The point of the exercise was to implement this function manually and not use one from a library 😅

Reply:
well, I implemented it manually first, noticed it wasn't that difficult and then decided to give up to my "don't rewrite code" instinct